### DISCUSSION TITLE
Unable to resolve module ...resolveAssetSource.js....

### DISCUSSION THREAD

**Discussion by moldender** [rnmapbox/maps#1624] [Q&A]
Hi guys!

I wanted to make a demo like thing with this package on snack.expo.dev but i'm always getting this error: 
```
Unable to resolve module 'react-native/Libraries/Image/resolveAssetSource.js'
  Evaluating react-native/Libraries/Image/resolveAssetSource.js
  Evaluating @react-native-mapbox-gl/maps.js
  Evaluating App.js
  Loading App.js
```

is there any way that i can use `react-native-mapbox-gl` in expo? Used with the latest versions

```
"@react-native-mapbox-gl/maps": "*"
"prop-types": "*"
```

**Comment by ferdicus** [comment]
Did you check out the installation guide? https://github.com/react-native-mapbox-gl/maps/blob/master/plugin/install.md?
